eclipse/sw360
# fix(escapeXML): Added missing escapeXML ,to prevent js script execution and rendering break due to single or double qoutes

> Added missing escapeXML ,to prevent js script execution and rendering break due to single or double qoutes
    * Added missing escapeXml to merge-split Component, merge releases, license details view, list-details-edit view of project, component and release.
> * Which issue is this pull request belonging to and how is it solving it? (*#907*, *#883*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: 

### How To Test?
> Add values listed below for different fields in Project, Component, Release, Vendor, License.
`'` 
`"`
`<script>alert('Testing script execution!')</script>`,
`" + function() { alert('Testing script execution!'); }() + " `
 `Navigate` to List view , Details view, Edit View of Project , Component, Release, Licenses , merge-split Component, merge Releases and `perform operation` like create-update-merge-split . Also try `adding Linked Release or Project` with special characters.
`Expected behavior:` Page should render properly and no script should be executed like alert box.

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
